### ISSUE TITLE
LaTeX printing does not work with IPython 4.0

### ISSUE THREAD

**Issue by jorisvandenbossche** [sympy/sympy#9838]
After upgrading IPython to 4.0, the latex pretty printing in the notebook does not work anymore.

E.g.

```
import sympy as sp
sp.init_printing()
x = sp.Symbol('x')
x**2
```
previously gave a latex output, but now just plaintext. Also doing explicitly `sp.init_printing(use_latex=True)` makes no difference.

Possibly related to #9799

**Comment by jorisvandenbossche** [comment]
This was due to a `ip.__module__.startswith('IPython')` which is now ipykernel', and already fixed by #9418! Sorry for the noise.

Are there plans to release a bugfix for this? (or just a new release in the near future?)

**Comment by moorepants** [comment]
@asmeurer This might warrant a bug fix release (or new one soon), as it will break for everyone that upgrades to IPython 4.0. Or we need to tell the IPython guys this and see if they have an solution from their end.
